Projekt dla ambitnych 1/Gruntfile.js

```diff
@@ -0,0 +1,68 @@
+module.exports = function(grunt) {
+
+  // Project configuration.
+  	grunt.initConfig({
+  		sass: {
+     		options: {
```

Comment:
`options:` is a child of `sass:`. so you should add one more tab to it, to show relationship